logstash-core/lib/logstash/config/pipelines_info.rb

```diff
@@ -13,6 +13,14 @@ def self.format_pipelines_info(agent, metric_store, extended_performance_collect
         p_stats = stats[pipeline_id]
         # Don't record stats for system pipelines
         next nil if pipeline.system?
+        i = 0
+        for output in p_stats[:plugins][:outputs]
+          plugin_id = output[i].to_s
+          if LogStash::PluginMetadata.exists?(plugin_id)
+            cluster_uuids = LogStash::PluginMetadata.for_plugin(plugin_id).get(:cluster_uuid)
```

Comment:
this local variable is assigned, but not aver accessed before falling out of scope, which leads me to believe that this entire hunk is not necessary.